src/openforms/registrations/contrib/zgw_apis/tests/test_backend.py

```diff
@@ -27,54 +31,11 @@ def setUpTestData(cls):
             ztc_service__api_root="https://catalogus.nl/api/v1/",
         )
 
-    def test_submission_with_zgw_backend(self, m):
-        submission = SubmissionFactory.from_components(
-            [
-                {
-                    "key": "voornaam",
-                    "registration": {
-                        "attribute": RegistrationAttribute.initiator_voornamen,
-                    },
-                },
-                {
-                    "key": "achternaam",
-                    "registration": {
-                        "attribute": RegistrationAttribute.initiator_geslachtsnaam,
-                    },
-                },
-                {
-                    "key": "tussenvoegsel",
-                    "registration": {
-                        "attribute": RegistrationAttribute.initiator_tussenvoegsel,
-                    },
-                },
-                {
-                    "key": "geboortedatum",
-                    "registration": {
-                        "attribute": RegistrationAttribute.initiator_geboortedatum,
-                    },
-                },
-            ],
-            submitted_data={
-                "voornaam": "Foo",
-                "achternaam": "Bar",
-                "tussenvoegsel": "de",
-                "geboortedatum": "2000-12-31",
-            },
-            bsn="111222333",
-        )
-
-        attachment = SubmissionFileAttachmentFactory.create(
-            submission_step=submission.steps[0],
-        )
-
-        zgw_form_options = dict(
-            zaaktype="https://catalogi.nl/api/v1/zaaktypen/1",
-            informatieobjecttype="https://catalogi.nl/api/v1/informatieobjecttypen/1",
-            organisatie_rsin="000000000",
-            vertrouwelijkheidaanduiding="openbaar",
-        )
+    def setUp(self):
+        # reset cache to keep request_history indexes consistent
+        schema_fetcher.cache.clear()
```

Comment:
doesn't hurt to also ensure it gets wiped at the end of the test:

`self.addCleanup(schema_fetcher.cache.clear)`